xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/CommentForm.java

```diff
@@ -39,6 +39,8 @@ public class CommentForm extends BaseElement
      */
     private By containerLocator;
 
+    private final By textareaLocator = By.tagName("textarea");
```

Comment:
I'm wondering if it would not make more sense to make this a static since it's always the same thing from what I understand, as any other cons variable.